src/main/java/gregapi/util/WD.java

```diff
@@ -134,7 +134,7 @@ public static boolean obstructed(World aWorld, int aX, int aY, int aZ, byte aSid
 		aX += OFFSETS_X[aSide]; aY += OFFSETS_Y[aSide]; aZ += OFFSETS_Z[aSide];
 		TileEntity tTileEntity = te(aWorld, aX, aY, aZ, T);
 		if (tTileEntity != null) {
-			if (tTileEntity instanceof ITileEntityQuickObstructionCheck) return ((ITileEntityQuickObstructionCheck)tTileEntity).isObstructingBlockAt(OPPOSITES[aSide]);
+			if (tTileEntity instanceof ITileEntityQuickObstructionCheck) return F;
```

Comment:
What exactly is this supposed to be? You could No-OP the entire function by generally returning false instead of keeping that instanceof check.